lib/griffin/interceptors/server/newrelic_interceptor.rb

```diff
@@ -20,25 +20,30 @@ def request_response(method: nil, call: nil, **)
           return yield if @ignored_services.include?(service_name)
 
           transaction_name = build_transaction_name(service_name, method.name)
-          state = NewRelic::Agent::TransactionState.tl_get
 
           # gRPC's HTTP method is fixed. https://github.com/grpc/grpc/blob/af89e8c00e796f3398b09b7daed693df2b14da56/doc/PROTOCOL-HTTP2.md
           req = Request.new("/#{transaction_name}", call.metadata['user-agent'], 'POST')
           # ":controller" is not correct category name for gRPC, But since we want to categorized this transaction as web transactions.
           # https://docs.newrelic.com/docs/apm/transactions/key-transactions/introduction-key-transactions
-          NewRelic::Agent::Transaction.start(state, :controller, transaction_name: "Controller/#{transaction_name}", request: req)
+          finishable = NewRelic::Agent::Tracer.start_transaction_or_segment(
```

Comment:
When was this method introduced?
We might have to bump up the minimum required version of newrelic_rpm to use this API.

Reply:
It was introduced in 6.0.0, and `NewRelic::Agent::Transaction#start` was also removed. 
https://github.com/newrelic/rpm/commit/c9a467d22e6d4b893be33be330d258a125cf813a#diff-e0d9e66301afac32aa77c6f4f58ad9a6L121-L135
We need to bump up the minimum version as you said.
I thought that we need it when we supports Ruby 2.7.
https://github.com/newrelic/rpm/blob/master/CHANGELOG.md#v690
It might be no back-ported.